src/main/resources/org/damap/base/db/changeLog-4.x/changeLog-4.0.0_4.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: Geoffrey Karnbach
+      changes:
+        - addColumn:
+            tableName: project
+            columns:
+              - column:
+                  name: acronym
+                  type: varchar(255)
+        - addColumn:
+            tableName: project_aud
+            columns:
+              - column:
+                  name: acronym
+                  type: varchar(255)
```

Comment:
a revert attribute would be great. This will allow to easily revert a change log

Reply:
Added